sklearn/utils/graph.py

```diff
@@ -119,18 +122,26 @@ def graph_laplacian(csgraph, normed=False, return_diag=False):
 
     if sparse.isspmatrix(csgraph):
         return _laplacian_sparse(csgraph, normed=normed,
-                                 return_diag=return_diag)
+                                 return_diag=return_diag, copy=copy)
     else:
         return _laplacian_dense(csgraph, normed=normed,
-                                return_diag=return_diag)
+                                return_diag=return_diag, copy=copy)
 
 
-def _laplacian_sparse(graph, normed=False, return_diag=False):
+def _laplacian_sparse(graph, normed=False, return_diag=False, copy=True):
     n_nodes = graph.shape[0]
-    if not graph.format == 'coo':
-        lap = (-graph).tocoo()
+    if not copy:  # use the same matrix
+        np.negative(graph, out=graph)
+        if not graph.format == 'coo':
+            lap = graph.tocoo(True)  # prevent making a new copy
```

Comment:
I think True makes a copy. To be clearer, use the parameter by name, i.e. `copy=False`